changelog.txt

```diff
@@ -1,5 +1,7 @@
 # To be released:
 
+- Update events fields related to read status
```

Comment:
It is a breaking change, because we are removing some fields. Could you detailed it?